wiki/Manual.md

```diff
@@ -2423,32 +2423,11 @@ If `since` is not specified the `fetchOHLCV` method will return the time range a
 
 ### Notes On Latency
 
-Trading strategies require fresh up-to-date information for technical analysis, indicators and signals. Building a speculative trading strategy based on the OHLCV candles received from the exchange may have critical drawbacks. Developers should account for the details explained in this section to build successful bots.
```

Comment:
I think the original text is a bit more professional and better structured. I don't see a reason to update it.

Reply:
@carlosmiei that commit didnt not really belong here, sorry, i don't understand how that commit was included.

the only change i intended for this PR was to remove one `#`, plz please update now

Reply:
i think i understand now, i wanted to create a new branch in vscode, and sometimes if you match keywords of existing branch, it doesnt create a new branch, instead autofills the suggested existing branch. something like that happened.

Reply:
please check also this PR, which was meant for OHLCV paragraph: https://github.com/ccxt/ccxt/pull/21587/files